src/main/java/uk/gov/pay/directdebit/gatewayaccounts/api/CreateGatewayAccountValidator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.directdebit.gatewayaccounts.api;
+
+import com.google.common.collect.ImmutableMap;
+import uk.gov.pay.directdebit.common.validation.ApiValidation;
+import uk.gov.pay.directdebit.common.validation.FieldSize;
+
+import java.util.Map;
+import java.util.function.Function;
+
+public class CreateGatewayAccountValidator extends ApiValidation {
+
+    private static final String PAYMENT_PROVIDER_KEY = "payment_provider";
+    private static final String SERVICE_NAME_KEY = "service_name";
+    private static final String TYPE_KEY = "type";
+
+    private static final int SERVICE_NAME_FIELD_LENGTH = 50;
```

Comment:
As discussed we should clarify the service name validation logic and possibly remove service name from direct debit connector